src/lib/openjp2/openjpeg.h

```diff
@@ -564,6 +564,7 @@ typedef struct opj_dparameters {
     /* <<UniPG */
 
     unsigned int flags;
+    unsigned int dump_state; /* AFL test */
```

Comment:
This will change the ABI. An alternative way would be to add a OPJ_DPARAMETERS_DUMP_FLAG 0x0002 flag (see existing OPJ_DPARAMETERS_IGNORE_PCLR_CMAP_CDEF_FLAG)